.github/workflows/reuse.yml

```diff
@@ -0,0 +1,254 @@
+name: If Levaithan was reusuable?
+
+on:
+  workflow_call:
+    inputs:
+      LEVIATHAN_REF:  # id of input
```

Comment:
nit, but usually inputs are lowercase and separated with hyphens ([example](https://github.com/actions/checkout/blob/main/action.yml))

Reply:
Best practices has to be looked after hard with this PR, I didn't care much about it writing these.